packages/ui/src/composites/wui-icon-link/styles.ts

```diff
@@ -7,12 +7,6 @@ export default css`
     padding: var(--wui-spacing-2xs);
   }
 
-  @media (max-width: 700px) {
```

Comment:
I think we should add this again. I tried it on mobile and had a few moments where I just missed the hitbox.